### ISSUE TITLE
Exporting a diagram with inline latex formulas without "dark" box checked leaves formulas unprocessed.

### ISSUE THREAD

**Issue by zajer** [jgraph/drawio-desktop#515]
### Preflight Checklist

* [x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio-desktop/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

**Describe the bug**
When I export a diagram with an inline latex formula in it to either jpg or png it isn't being rendered if I won't check "dark" box while exporting.
**To Reproduce**
1. Make a diagram with an inline latex formula in it.
2. Go to 'Export as'
3. Click on either 'PNG' or 'JPEG'
4. Uncheck 'Dark' checkbox
5. Click 'Export'
6. See the exported image.

**Expected behavior**
The exported image should contain inline latex formula in "plain text" i.e. not transformed into a proper math notation.
This behavior does not occur if we don't uncheck the "Dark" checkbox during the exportation process.
**Screenshots**
Without the "Dark" checkbox checked:
![test](https://user-images.githubusercontent.com/13221275/109376173-174fe580-78c3-11eb-849a-459a81b13a04.png)
With the "Dark" checkbox checked:
![test-d](https://user-images.githubusercontent.com/13221275/109376216-60079e80-78c3-11eb-9be0-0bc833d85ac9.png)

**draw.io version (In the Help->About menu of the draw.io editor):**

- draw.io version : 14.4.3
**Desktop (please complete the following information):**
 - OS: Windows 10

**Additional context**
Don't know where to put so I will write it here.
Thank you for amazing work with this project. This is really a very useful and well written piece of software.

**Comment by mararad** [comment]
Reproduced in desktop draw.io 14.4.3 and online 14.4.3 in Chrome incognito, Win 10 Pro

**Comment by mararad** [comment]
Another export issue from dark theme but export as SVG 
https://github.com/jgraph/drawio-desktop/issues/693

1. Go to the app.
2. Create a diagram using LaTeX.
3. File -> Export As -> SVG...
4. Change Theme to different -> Export -> Save
5. See the $$ signs.

I reproduced it in desktop 14.9.6 and online 15.1.0 in Chrome Incognito (Win 10)

**Comment by SpacewaIker** [comment]
I've encountered the same issue when exporting as "Advanced...". In fact, with the advanced option, there is no checkbox for dark mode so the latex/mathematical typesetting is always converted back into plain text. (desktop version 16.5.1 on Win11)

A workaround I've found is to first export as PDF, then take a screenshot of the PDF to get a PNG. However this is pretty sketchy and it's not possible to get a transparent background either.
